README.md

```diff
@@ -1,31 +1,14 @@
 # SAGED
 
 ![Status Badge](https://github.com/greenelab/saged/workflows/PythonTests/badge.svg)
-[![Coverage Status](https://coveralls.io/repos/github/ben-heil/saged/badge.svg?branch=master)](https://coveralls.io/github/ben-heil/saged?branch=master)
```

Comment:
Do you want some kind of description in the README of what you're doing? Or would you add that later?

Reply:
I definitely will, at some point in the future I'm going to do a big housekeeping PR where I rewrite the README and remove unused code/files